src/app.html

```diff
@@ -6,7 +6,7 @@
 		<meta name="viewport" content="width=device-width" />
 		%sveltekit.head%
 	</head>
-	<body data-sveltekit-preload-data="hover" class="h-full overflow-hidden" data-theme="">
+	<body data-sveltekit-preload-data="hover" class="h-full overflow-hidden" data-theme="my-custom-theme">
```

Comment:
dont hardcode

Reply:
see hooks.server.ts and change that from modern to this, also change it's name from my-custom-theme to weeb-crimson or smthn like that